Vagrantfile

```diff
@@ -3,10 +3,11 @@ Vagrant.configure("2") do |config|
     config.vm.box_version = ">= 0.3, < 0.4"
     config.vm.provider "virtualbox" do |vb|
         vb.linked_clone = true
-        vb.memory = 2048
+        vb.memory = 10000
+        vb.cpus = 4
```

Comment:
This should be 3 cpus as many machines have 4 cpus, and you will need one to keep vbox and os running.

Reply:
Good point